stdlib/asyncio/__init__.pyi

```diff
@@ -85,18 +87,16 @@ if sys.version_info >= (3, 8):
         TimeoutError as TimeoutError,
     )
 else:
-    if sys.version_info >= (3, 7):
-        from .events import SendfileNotAvailableError as SendfileNotAvailableError
     from .futures import CancelledError as CancelledError, InvalidStateError as InvalidStateError, TimeoutError as TimeoutError
     from .streams import IncompleteReadError as IncompleteReadError, LimitOverrunError as LimitOverrunError
 
-if sys.version_info >= (3, 7):
-    from .protocols import BufferedProtocol as BufferedProtocol
+    if sys.version_info >= (3, 7):
+        from .events import SendfileNotAvailableError as SendfileNotAvailableError
```

Comment:
It took me a while to realize how `SendfileNotAvailableError` works. On Python 3.8 or newer, it is imported from `.exceptions`, along with other things. On Python 3.7, it is imported from `.events` instead. It would be nice to give it its own `if`,`else` statements, just to make this very clear.

Reply:
Yeah, that makes sense.